### ISSUE TITLE
Paper UI needs a manual refresh to show bridge on Configuration / Things

### ISSUE THREAD

**Issue by MHerbst** [eclipse/smarthome#3391]
Tested with a fresh installation of openHAB Build 909 on Windows 10 with Chrome 58.

I performed the following steps:

- Page "Inbox": "Search for things"
- Selected "Homematic Binding"
- Selected "Add manually" and again "Homematic Binding"
- Configured Homematic Bridge and saved the configuration
- Browser automatically opens page "Configuation > Things", which is empty:
![image](https://cloud.githubusercontent.com/assets/616792/25783667/a66bbd9a-3360-11e7-8a39-6a3aa7421b77.png)

After a manual browser refresh the bridge is shown:
![image](https://cloud.githubusercontent.com/assets/616792/25783670/b9d0649e-3360-11e7-955a-8cffa540476f.png)

**Comment by kaikreuzer** [comment]
@aounhaider1 Could you please check?

**Comment by aounhaider1** [comment]
I am unable to reproduce this issue. It seems to be working fine. Is this completely not working for you? A possible problem could be the loss of event stream. PaperUI tries to reestablish the connection ASAP but updates, such as add/remove thing, are lost in this case.

**Comment by MHerbst** [comment]
@aounhaider1 I can reproduce it anytime with the Homematic binding. But I think it only occurs for bridge things that are created indirectly.
I have captured the steps I did with a screen record. You can watch the video here: https://drive.google.com/file/d/0B96-rrUfIcUWOWp5UGVGWjkwdTQ/view

Maybe you can try it also with this binding. It doesn't matter whether the binding can connect to a CCU or not. I could reproduce it with successful initialization and also with connection errors.

**Comment by htreu** [comment]
I downloaded the latest build 912 and tested on Chrome 58 (macOS) and can **not** reproduce the issue. After manually adding the Homematic Bridge with a fake ip address the bridge shows up immediately. Also checked Safari 10.1 and works there, too.

**Comment by MHerbst** [comment]
I did some more tests with build 912 and I found the cause of the problem: it's the ESET firewall on my Windows machine. If I disable the firewall everything works fine. With an enabled firewall, I can reproduce the problem.
It makes no difference where openHAB is installed. With an activated firewall, it also happens if openHAB is installed on the same machine.
I will try to figure out which firewall setting causes this problem. This issue can be closed.
